wcfsetup/install/files/acp/templates/userList.tpl

```diff
@@ -56,7 +56,7 @@
 
 <header class="contentHeader">
 	<div class="contentHeaderTitle">
-		<h1 class="contentTitle">{lang}{@$pageTitle}{/lang}
+		<h1 class="contentTitle">{lang}{@$pageTitle}{/lang}</h1>
```

Comment:
I thought I shall remove it because there always are some users. I shouldn't be that hectic. Sorry.